how-to-docs/namespace-how-to.md

```diff
@@ -1,5 +1,5 @@
 ## How do I create a namespace in the UK cluster?
-Please visit the [ACP-Kube-Namespaces](https://gitlab.digital.homeoffice.gov.uk/Devops/acp-kube-namespaces) repo on Gitlab to create a namespace.
+Please visit the [ACP-Kube](https://gitlab.digital.homeoffice.gov.uk/Devops/acp-kube) repo on Gitlab to create a namespace.
 
 + Clusters are defined by the folders within the `namespaces/` directory. This directory contains all defined namespaces for each given cluster.
```

Comment:
"defined by the folders within the `policies/` directory, with each cluster having a subdirectory labelled `namespaces`" - Or something along those lines